### DISCUSSION TITLE
Ordner erstellen

### DISCUSSION THREAD

**Discussion by Hoddl666** [jonaswinkler/paperless-ng#747] [Feature Requests]
Hallo,

eine Ordner Struktur wäre gut z.b.

Hauptordner
      Unterordner
           Unterordner

**Comment by jonaswinkler** [comment]
> Hallo,
> 
> eine Ordner Struktur wäre gut z.b.
> 
> Hauptordner
> Unterordner
> Unterordner

English transcription

> Hello,
>
> A folder structure would be nice, for example
>
> Main folder
> Sub folder
> Sub folder

I'll not add folders to paperless, since tags are vastly superior to folders. A document can only ever appear in one folder. Think of tags as folders, but a single document can appear in multiple folders. If you have two folders "tax 2020" and "work", where would you put a document that belongs to both?
